extension/script/common/socket.lua

```diff
@@ -119,12 +119,12 @@ return function (param)
             session:close()
             fds[1] = session
         elseif t.mode == "listen" then
-            server:close()
-            fds[1] = server
-            if session ~= nil then
-                session:close()
-                fds[2] = session
-            end
+			fds[1] = server
+			if session ~= nil then
+				session:close()
+				fds[2] = session
+			end
+			server:close()
```

Comment:
What do these changes do?

Reply:
This is optional. Just conceptually, clossing the session before shutting down the server.